frontend/src/locales/en/models.json

```
@@ -264,7 +265,12 @@
   "models.developers.stepfun": "StepFun",
   "models.developers.xai": "xAI",
   "models.developers.xiaomi": "Xiaomi",
+  "models.developers.longcat": "LongCat",
+  "models.developers.nvidia": "NVIDIA",
+  "models.developers.bytedance": "ByteDance",
   "models.developers.zai": "Z.ai",
+  "models.developers.ibm": "IBM",
+  "models.developers.inclusionai": "Inclusion AI",
```

Biome (2.5.8)

[error] The key models.developers.inclusionai was already declared.

(lint/suspicious/noDuplicateObjectKeys)